documentation/source/quickstart.rst

```diff
@@ -178,8 +178,9 @@ We would then create a file called ``test_my_design.py`` containing our tests.
 Creating a test
 ---------------
 
-The test is written in Python.  Assuming we have a toplevel port called ``clk``
-we could create a test file containing the following:
+The test is written in Python. Cocotb wraps your top level with the handle **dut**
```

Comment:
Worth noting that the argument name doesn't matter - it's just passed as the first argument

Reply:
if it doesn't have to be dut, that's definitely worth mentioning. I couldn't tell if it was required, but didn't see any examples that didn't use dut either.